main.js

```diff
@@ -0,0 +1,70 @@
+
+
+const id1 = new APIManager()
```

Comment:
would be better to call it `const apiManager = newAPIManager()`
just because `id1` is shorter does not mean it is better

It makes reading code a lot harder if you do not name variables what they are.